.gitignore

```diff
@@ -58,3 +58,4 @@ props-table
 **/__snapshots__/screen
 **/__snapshots__/diff
 errorScreenshots
+.DS_Store
```

Comment:
Should we do this with folder structure to account for other files having them?

    */.DS_Store/

Reply:
Talked offline with @JakeLaCombe and @msaikaushik1992, we can leave this as is. This will apply recursively though the directories.